modules/api/src/main/java/org/apache/ignite/configuration/schemas/network/NetworkConfigurationSchema.java

```diff
@@ -23,18 +23,20 @@
 import org.apache.ignite.configuration.annotation.Value;
 import org.apache.ignite.configuration.storage.ConfigurationType;
 
-/**
- * Configuration schema for network endpoint subtree.
- */
 @ConfigurationRoot(rootName = "network", type = ConfigurationType.LOCAL)
 public class NetworkConfigurationSchema {
-    /** */
+    /** Uniq local node name. */
+    @Value(hasDefault = true)
+    public String name = "";
+
+    /**
```

Comment:
Should we use /** */ according to our guidelines? If true please fix here and below.

Reply:
Done.